src/commands/app/logs.js

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const openwhisk = require('openwhisk');
+const aioConfig = require('@adobe/aio-lib-core-config')
+
+const { flags } = require('@oclif/command')
+// const { cli } = require('cli-ux')
+const BaseCommand = require('../../BaseCommand')
+
+class Logs extends BaseCommand {
+  async run () {
+    const { flags } = this.parse(Logs)
+
+    this.aioConfig = aioConfig.get() || {}
+    await this.getLogs(flags.limit)
+    this.log(`✔ Finished fetching logs!`)
+  }
+
+  async getLogs (limit) {
+    const auth = Buffer.from(this.aioConfig.runtime.auth).toString('base64')
+    const authHandler = {
+      getAuthHeader: ()=>{
+        return Promise.resolve('Basic ' + auth)
+      }
+    }
+    // get activations
+    const options = {apihost: 'runtime.adobe.io',
+    auth_handler: authHandler,
```

Comment:
you can directly pass the auth through `api_key` param

Reply:
done